doc/source/ray-overview/getting-started.md

```diff
@@ -610,10 +598,9 @@ Learn more about Ray Core
 
 ``````
 
-## Ray Cluster Quick Start
+## Ray Cluster Quickstart
 
-You can deploy your applications on Ray clusters, often with minimal code changes to your existing code.
-See an example of this below.
+Deploy your applications on Ray clusters, often with minimal code changes to your existing code.
```

Comment:
I just realized that this quickstart...

1. is missing steps.
2. has a cluster config that is way too complicated.

See https://docs.ray.io/en/master/cluster/vms/getting-started.html

Reply:
yikes! ok - I'll capture this in a new GH issue.